server/src/test/java/org/elasticsearch/index/shard/DenseVectorStatsTests.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.index.shard;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.test.AbstractWireSerializingTestCase;
+
+public class DenseVectorStatsTests extends AbstractWireSerializingTestCase<DenseVectorStats> {
+    @Override
+    protected Writeable.Reader<DenseVectorStats> instanceReader() {
+        return DenseVectorStats::new;
+    }
+
+    @Override
+    protected DenseVectorStats createTestInstance() {
+        DenseVectorStats stats = new DenseVectorStats(randomNonNegativeLong());
+        return stats;
+    }
+
+    @Override
+    protected DenseVectorStats mutateInstance(DenseVectorStats instance) {
+        return new DenseVectorStats(randomNonNegativeLong());
```

Comment:
`randomValueOtherThan(instance.getValueCount(), ESTestCase::randomNonNegativeLong)`

Reply:
Trivially, I don't see that this has been resolved - but of course it's a very narrow potential for failure :-)

Reply:
I implemented the change on my local copy, but there was a hiccup with the push. I've rectified the situation now, so everything should be resolved. Thank you for double checking!